### DISCUSSION TITLE
Immortal doesnt show rank!

### DISCUSSION THREAD

**Discussion by erichl4** [ValveSoftware/Dota2-Gameplay#3857] [Q&A]
![image](https://user-images.githubusercontent.com/115933310/196037712-70d44bbb-48ff-4da8-a310-05a0572c8950.png)

So I'm in immortal rank for a while now and have played 11-12 games in immortal game, but the ranking hasn't updated yet even after 10+ games. One of my friend got the immortal ranking number after one game and his MMR is only 5640... while mine rn is 5780MMR

MMR Recalibration is not available also since i have used it already 5-6 months ago

**Comment by onesmartsheep1** [comment]
I am facing a similar issue. If anyone have found an answer or cure, or even the cause of the glitch, please let me know[.](https://onesmartsheep.com/) Thank you in advance.

**Reply by erichl4** [reply]
So this is my post, and i recently got my number now.  I think the reason my number doesn't pops off is because i didn't play 14 matches within 21 days, basically, you should try to play maybe 10+ game within 21 days.
